cmd/govim/config/config.go

```diff
@@ -163,6 +163,19 @@ type Config struct {
 	// GOFLAGS=-modfile=go.local.mod in order to use an alternative go.mod file.
 	GoplsEnv *map[string]string `json:",omitempty"`
 
+	// GoplsDirectoryFilters can be used to exclude unwanted directories from the
+	// workspace. By default, all directories are included. Filters are an
+	// operator, `+` to include and `-` to exclude, followed by a path prefix
+	// relative to the workspace folder. They are evaluated in order, and
+	// the last filter that applies to a path controls whether it is included.
+	// The path prefix can be empty, so an initial `-` excludes everything.
+	//
+	// Examples:
+	// Exclude node_modules: `-node_modules`
```

Comment:
Not specific to your PR, rather the choice on the `gopls` side. Is it possible to ignore any `node_modules` directory, regardless of where it appears?

I would have thought the [`.gitignore` approach](https://git-scm.com/docs/gitignore) might have been better for this reason.

Reply:
Will follow up on this question in a `gopls` issue.